apps/src/aiComponentLibrary/suggestedPrompt/SuggestedPrompts.tsx

```diff
@@ -20,28 +19,35 @@ interface SuggestedPromptsProps {
 const SuggestedPrompts: React.FC<SuggestedPromptsProps> = ({
   suggestedPrompts,
 }) => {
-  const setValues = (selected: string[]) => {
-    suggestedPrompts.map(prompt => {
-      if (prompt.selected !== selected.includes(prompt.label)) {
-        prompt.onClick(prompt);
-      }
-    });
-  };
+  const hasSelection = suggestedPrompts.some(prompt => prompt.selected);
+  const visiblePrompts = suggestedPrompts.filter(prompt => prompt.show);
 
   return (
-    <Chips
-      options={suggestedPrompts.flatMap(prompt =>
-        prompt.show ? {label: prompt.label, value: prompt.label} : []
-      )}
-      values={suggestedPrompts.flatMap(prompt =>
-        prompt.selected ? [prompt.label] : []
-      )}
-      setValues={setValues}
-      name={'Suggested Prompts'}
-      size={'s'}
-      textThickness={'thick'}
+    <div
       className={moduleStyles.prompts}
-    />
+      role="group"
+      aria-label="Suggested Prompts"
+    >
+      {visiblePrompts.map(prompt => {
+        const isDisabled = hasSelection && !prompt.selected;
+
+        return (
+          <button
+            key={prompt.label}
+            type="button"
+            className={`${moduleStyles.prompt} ${
+              prompt.selected ? moduleStyles.selected : ''
+            } ${isDisabled ? moduleStyles.disabled : ''}`}
```

Comment:
Nit: this can be cleaned up and easier to read with `classNames`:
```suggestion
            className={classNames(
              moduleStyles.prompt,
              prompt.selected && moduleStyles.selected,
              isDisabled && moduleStyles.disabled
            )}
```

Reply:
Okay this is silly but Claude is telling me the version I updated it to is a little bit preferred, but I used classNames!

Reply:
Oh Claude why! 🙃

Reply:
lolol It was saying that using the && means that it returns 'false' when the conditions aren't true. And that even though classNames filters out falsey values, this way is slightly cleaner. But hey I'm down to let our Claude agents battle😅

Reply:
I respectfully disagree Claude 😂

Reply:
okay wait but with the new one too?

Reply:
The new Claude? It's definitely cleaner and easier to read using classNames, and that's the convention we use elsewhere so feel free to ignore Claude in this case.

Reply:
No the new code- I updated it after you commented with classNames just using a different convention!

Reply:
I agree that I prefer classNames in general.